### PR TITLE
Fix Subnet Update Issue

### PR DESCRIPTION
**Pull request by NavisJ**
# Description
Fix Subnet Update Issue of Groupnet Rename

# ISSUE TYPE
- Bugfix Pull Request

##### RESOURCE OR DATASOURCE NAME
Subnet 

##### OUTPUT

image

##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Unit tests
- [x] Acceptance tests

### CODE REVIEW

**Comment by NavisJ** [comment]
run e2e test

File: powerscale/provider/subnet_resource.go [hunk 1]
```diff
@@ -102,8 +102,8 @@ func (r *SubnetResource) Schema(ctx context.Context, req resource.SchemaRequest,
 				Computed:            true,
 			},
 			"groupnet": schema.StringAttribute{
-				Description:         "Name of the groupnet this subnet belongs to.",
-				MarkdownDescription: "Name of the groupnet this subnet belongs to.",
+				Description:         "Name of the groupnet this subnet belongs to. Cannot update",
```
Comment (baoy1) [inline powerscale/provider/subnet_resource.go:105]: Just thinking if `Updating is not allowed.` is better wording...

**Review by baoy1 — APPROVED** [review]

**Review by RayLiu7 — APPROVED** [review]

**Comment by NavisJ** [comment]
run e2e test

**Comment by forrestxia** [comment]
run e2e test

**Pull request merged**